<!-- ... -->
<launch>
  <arg name="pi/2" value="1.5707963267948966" />
  <arg name="optical_rotate" value="0 0.06 0 0 0 0 1" />

  <node pkg="avc_vision" type="ocam" name="ocam" output="screen">
    <param name="resolution"    value="2"/> <!-- 0: 1280x960, 1: 1280x720, 2: 640x480, 3: 640x360 -->
    <param name="frame_rate"    value="30"/>
    <param name="exposure"      value="100"/>
    <param name="gain"          value="50"/>
    <param name="wb_blue"       value="200"/>
    <param name="wb_red"        value="160"/>
    <param name="auto_exposure" value="true"/>
    <param name="show_image"    value="false"/>

    <param name="left_frame_id" value="left_frame"/>
  </node>


</launch>
